SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -35,7 +35,7 @@ namespace
 {
 const FString DEFAULT_WORKER_REGION_MATERIAL =
 	TEXT("/SpatialGDK/SpatialDebugger/Materials/TranslucentWorkerRegion.TranslucentWorkerRegion");
-}
+} // namespace
```

Comment:
Does running clang-format not do this automatically?
```suggestion
} // anonymous namespace
```

Reply:
Doesn't seem to. Didn't do it for the enum namespace either.